pkg/backend/display/rows.go

```diff
@@ -282,6 +282,13 @@ func (data *resourceRowData) IsDone() bool {
 		return true
 	}
 
+	if isRootStack(data.step) {
```

Comment:
For my own understanding, what are the cases where `data.ContainsOutputsStep(data.step.Op)` returns "true" but the stack is not actually done updating?  I would have assume that we would not see the outputs until the entire stack has completed updating?

Reply:
Indeed.  This is a mitigation until the root cause of what you're describing is understood.  This shouldn't be happening but it is.  THis is an easy way to get back to acceptable behavior while a deeper investigation is done.